randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -279,13 +281,22 @@ Extra - asset_id: 76
   * Layers: default
   * Normal Door to Etecoon Save Room/Door to Etecoon Energy Tank Room
   > Door to Green Brinstar Main Shaft
-      Morph Ball
+      Any of the following:
+          Morph Ball
+          # Tunnel Crawl: https://vimeo.com/800715640
```

Comment:
our video db does not go well with vimeo links

Reply:
Am I just supposed to not use vimeo links? Would we rather there be no video at all? Because it would be a monumental amount of work to re-record all of these videos.

Reply:
cant you just yt-dlp the links, reupload them to youtube, and give credits to the vimeo author?